MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -11,6 +11,11 @@ static partial class GamePad
     {
         internal static bool Back;
 
+        private static int PlatformGetMaxIndex()
+        {
+            return 16;
+        }
+
         private static GamePadCapabilities PlatformGetCapabilities(int index)
         {
             if (index >= WGI.Gamepad.Gamepads.Count)
```

Comment:
Also might be good to remove these index checks in the platform methods here.  Let the assert at the shared code level deal with the bounds check.